Core/src/main/java/com/mychefassistant/core/domain/Kitchen.kt

```diff
@@ -1,8 +1,10 @@
 package com.mychefassistant.core.domain
 
+import com.mychefassistant.core.utils.KitchenIcons
+
 data class Kitchen(
     val id: Int = 0,
     val title: String,
-    val icon: Int? = null,
+    val icon: KitchenIcons,

```

Comment:
You can instead store the type the deduce icon from type. Type is more general and can be used to achieve different goals.